shared-module/epaperdisplay/EPaperDisplay.h

```diff
@@ -26,7 +26,6 @@
 
 #pragma once
 
-#include "shared-bindings/digitalio/DigitalInOut.h"
```

Comment:
I think this is causing build errors.

Reply:
yeah this part of the change isn't right. Instead I need to figure out why EPaperDisplay was being included where it wasn't enabled.